.github/workflows/php-tests.yml

```diff
@@ -0,0 +1,70 @@
+name: PHP Tests
+
+on:
+  push:
+    branches:
+      - develop
+      - master
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - ready_for_review
+
+jobs:
+  php-tests:
+    name: Run PHP Tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php: ['5.6', '7.4']
+        wordpress: ['4.7', 'latest-stable', 'latest-stable (multisite)', 'nightly']
```

Comment:
We should be able to use versions here that are passed to the test install script.

This should only be `latest` and `nightly`. 4.7 and latest with multisite should be added via `include`.

`multisite` itself would be part of the matrix and be set like `env: WP_MULTISITE: {{ matrix.multisite }}` (compressed for brevity). See a similar example on the other project.